lib/install/management/appliance.go

```diff
@@ -902,9 +906,14 @@ func (d *Dispatcher) CheckDockerAPI(conf *config.VirtualContainerHostConfigSpec,
 					// This will occur if the server certificate was signed by a CA that is not the one used for client authentication
 					// and does not have a trusted root registered on the system running vic-machine
 					// This is a legitimate deployment so no error, but definitely requires a warning.
-					log.Warnf("Unable to verify server certificate with configured CAs: %s", neterr.Error())
+					log.Warnf("Unable to validate server certificate with configured CAs (unknown authority): %s", neterr.Error())
```

Comment:
I think this is only a warning if `tlsErrExpected` is true, but an error otherwise. If we get this, but thought we knew the correct address to use then it's an error.